packages/atclient/include/atclient/atkeys.h

```diff
@@ -13,12 +13,14 @@
 #define ATCLIENT_ATKEYS_ENCRYPT_PUBLIC_KEY_INDEX 0
 #define ATCLIENT_ATKEYS_ENCRYPT_PRIVATE_KEY_INDEX 0
 #define ATCLIENT_ATKEYS_SELF_ENCRYPTION_KEY_INDEX 0
+#define ATCLIENT_ATKEYS_ENROLLMENT_ID_INDEX 0
 
 #define ATCLIENT_ATKEYS_PKAM_PUBLIC_KEY_INITIALIZED (VALUE_INITIALIZED << 0)
 #define ATCLIENT_ATKEYS_PKAM_PRIVATE_KEY_INITIALIZED (VALUE_INITIALIZED << 1)
 #define ATCLIENT_ATKEYS_ENCRYPT_PUBLIC_KEY_INITIALIZED (VALUE_INITIALIZED << 2)
 #define ATCLIENT_ATKEYS_ENCRYPT_PRIVATE_KEY_INITIALIZED (VALUE_INITIALIZED << 3)
 #define ATCLIENT_ATKEYS_SELF_ENCRYPTION_KEY_INITIALIZED (VALUE_INITIALIZED << 4)
+#define ATCLIENT_ATKEYS_ENROLLMENT_ID_INITIALIZED (VALUE_INITIALIZED << 5)
```

Comment:
Variables to define the position of enrollment id in our is_initialized boolean vector